<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
  <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户注册</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelUser">
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="usernameInput"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelPass">
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="passwordInput">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelConfirm">
       <property name="text">
        <string>确认密码：</string>
       </property>
      </widget>
     </item>
    <item row="2" column="1">
      <widget class="QLineEdit" name="confirmInput">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
   <item row="3" column="0">
     <widget class="QLabel" name="labelPhone">
      <property name="text">
     <string>手机号（选填）：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="phoneInput"/>
    </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelEmail">
    <property name="text">
     <string>邮箱（选填）：</string>
    </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="emailInput"/>
   </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="submitButton">
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>RegisterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>submitButton</sender>
   <signal>clicked()</signal>
   <receiver>RegisterDialog</receiver>
   <slot>on_submitButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <resources/>
 <connections/>
</ui>
